docs/EN/operate/build.md

```diff
@@ -1,217 +1,261 @@
+
+#### xDrip4iOS
 
 ```
 /bin/bash -c "$(curl -fsSL \
-  https://raw.githubusercontent.com/loopnlearn/loopbuildscripts/main/BuildGlucoseDirect.sh)"
+  https://raw.githubusercontent.com/loopandlearn/lnl-scripts/main/BuildxDrip4iOS.sh)"
 ```
 
-The download is placed in your `Downloads` folder in a directory called `BuildxDrip4iOS` or `BuildGlucoseDirect` respectively. The downloaded clone is found in a folder with the branch name, date and time encoded.
-
-
-### Alternative Branch
-
-Sometimes, specific branches are offered for testing. Any desired branch can be cloned using the **Build_iAPS** script. After the final quote of the script command, add a space, hyphen, space and branch_name. An example is shown below; replace branch_name with your desired branch. Note that specific branches like this are not deleted as part of the `Delete Old Downloads` utility discussed in [Maintenance Utilities](#maintenance-utilities).
+#### Glucose Direct
 
 ```
 /bin/bash -c "$(curl -fsSL \
-  https://raw.githubusercontent.com/loopnlearn/loopbuildscripts/main/Build_iAPS.sh)" - branch_name
+  https://raw.githubusercontent.com/loopandlearn/lnl-scripts/main/BuildGlucoseDirect.sh)"
 ```
 
-### Maintenance Utilities
-
-Several maintenance utilities are availble with the Build Select Script which is typically used with Loop. For more information, refer to [Loop and Learn: Build Select Script](https://www.loopandlearn.org/build-select/) documentation.  Issue the command below in your terminal, select Option 3 for Maintenance Utilities. Then choose from these options:
+### Alternative Branch
 
-1. Delete Old Downloads
-1. Clean Derived Data
-1. Xcode Cleanup (The Big One)
-1. Return to Menu
+Sometimes, specific branches are offered for testing. Any desired branch can be cloned using the **Build Open-iAPS** script. After the final quote of the script command, add a space, hyphen, space, and branch_name. An example is shown below: replace `branch_name` with your desired branch. Note that specific branches like this are not deleted as part of the `Delete Old Downloads` utility discussed in [Maintenance Utilities](#maintenance-utilities).
 
 ```
 /bin/bash -c "$(curl -fsSL \
-  https://raw.githubusercontent.com/loopnlearn/loopbuildscripts/main/BuildSelectScript.sh)"
+  https://raw.githubusercontent.com/loopandlearn/lnl-scripts/main/BuildOpen-iAPS.sh)" - branch_name
 ```
 
-Once each option completes, you are returned to the top menu, so must select Maintenance Utilities again if desired.
-
 ## Update Open-iAPS with Xcode
 
 The recommendation is to use the Xcode Source Control features to update your code.
 
-If you prefer to use the command line interface, skip ahead to [Update Open-iAPS with CLI](#update-iaps-with-cli).
+If you prefer the command line interface, skip ahead to [Update Open-iAPS with CLI](#update-open-iaps-with-cli).
 
 ### Update Open-iAPS with Source Control
```

Comment:
@bjornoleh 
This entire section is out of date. Perhaps remove for now.
The Source Control menu is found under View->Navigators->Source Control

And because the new code consists of a workspace with submodules, not sure how the update process is supposed to work.